readme.md

```diff
@@ -28,6 +28,46 @@ oden_print file.pdf
 
 Use `oden_print -h` for a full help message.
 
+### Frequently used printing options
+
+To specify printing options supported by `lpr`, make use of the `-t` flag, followed by a string of options surrounded by quotation marks. Note that a `-o` is needed before each option.
+
+```
+# Passing lpr options through oden_print
+oden_print -t "-o <option1> -o <option2> ..." file.pdf 
+```
+
+Check out this [official CUPS documentation](https://www.cups.org/doc/options.html) page for a complete list of standard printing options. The options are listed at the bottom of the page. Here, we list a couple of options that are frequently used by many users.
```

Comment:
I think we should also suggest `man lpr`. I've found everything I've needed there, and if they're on the CLI it will be much faster.

Also, for the last sentence I prefer:
Here we review some of the most common options.

Reply:
Suggesting `man lpr` is great! I haven't thought of it myself, I just default to searching instructions online. 

However, I am quite sure `lpr` is not preloaded on at least Windows 10 (through PowerShell or CMD). I don't know if that is still the case on Windows 11. For windows users, the best way to obtain this information is going to be that webpage, if they do not `ssh` into their Oden workstations.

Feel free to change the language as you see fit. I sometimes still struggle with how to write things properly in English :(